great_expectations/expectations/core/expect_column_quantile_values_to_be_between.py

```diff
@@ -131,16 +133,81 @@ class ExpectColumnQuantileValuesToBeBetween(ColumnExpectation):
     success_keys = (
         "quantile_ranges",
         "allow_relative_error",
+        "auto",
+        "profiler_config",
     )
+
+    default_profiler_config: RuleBasedProfilerConfig = RuleBasedProfilerConfig(
+        name="expect_column_quantile_values_to_be_between",  # Convention: use "expectation_type" as profiler name.
+        config_version=1.0,
+        variables={
+            "quantiles": [
+                0.25,
+                0.5,
+                0.75,
+            ],
+            "allow_relative_error": "linear",
+            "num_bootstrap_samples": 9999,
+            "false_positive_rate": 0.05,
+            "truncate_values": {
+                "lower_bound": None,
+                "upper_bound": None,
+            },
+            "round_decimals": 1,
+        },
+        rules={
+            "column_quantiles_rule": {
```

Comment:
are there types we can use here, too?

Reply:
@donaldheppner Not yet -- for now, we follow exactly the same pattern for "Base" level business objects' configuration objects as established in `DataContextConfig` and `CheckpointConfig`.  Potentially can be done in the future, but a bit too many changes if attempted to do here.

Reply:
Agreed, I've added a task to the sprint for a refactor of this config type.